tests/setup/mod.rs

```diff
@@ -273,12 +337,49 @@ pub mod nigiri {
         Ok(address)
     }
 
-    pub fn lnd_node_open_channel(
+    pub fn get_cln_node_funding_address(node: NodeInstance) -> Result<String, String> {
+        let sub_cmd = &["newaddr"];
+        let cmd = [get_lnd_node_prefix(node), sub_cmd].concat();
+
+        let output = exec(cmd.as_slice());
+        if !output.status.success() {
+            return Err(produce_cmd_err_msg(&cmd, output));
+        }
+        let json: serde_json::Value =
+            serde_json::from_slice(&output.stdout).map_err(|_| "Invalid json")?;
+        let address = json["bech32"].as_str().unwrap().to_string();
+        Ok(address)
+    }
+
+    pub fn node_connect(
```

Comment:
I'd kindly disagree: https://github.com/getlipa/lipa-lightning-lib/blob/fdc39b69f43595784ac9538a12fb3dd2e046724f/tests/receiving_payment_test.rs#L294

Should I make it more obvious that the method actually works for `LND` AND `CLN`? Or should we actually ship a method for LND and one for CLN, because they should in fact be handled separately, resp. one of the instances could change their interface while the other wouldn't?

Reply:
If the method works for both then it's good. I thought the current implementation only works for LND given that CLN is documented as expecting `connect id [host] [port]` instead of `connect id@[host]:[port]` ([docs](https://lightning.readthedocs.io/lightning-connect.7.html))

Reply:
Oh okay. It does work for both. Going to resolve the issue.